site/src/components/pages/home/Header/index.tsx

```diff
@@ -0,0 +1,242 @@
+import React, { useEffect } from "react";
+import {
+  Container,
+  Typography,
+  Box,
+  useTheme,
+  useMediaQuery,
+} from "@mui/material";
+import { gsap } from "gsap";
+import { Button } from "../../../Button";
+import { BoltIcon } from "../../../SvgIcon/BoltIcon";
+import { Link } from "../../../Link";
+import { HeaderCard } from "./HeaderCard";
+
+export const Header = () => {
```

Comment:
During page-load the animation is displayed in its final sate, ideally it shouldn't do this:

https://user-images.githubusercontent.com/42802102/148939573-28e93eda-767b-4b30-8bf5-c6cb050a261c.mov

This is likely going to be more noticeable with slower internet connections. If this is an easy fix I'd suggest it's worth doing as part of this PR, if not let's create a follow-up task.

Reply:
Also noticed for wider screensizes the header card looses its fade on the right:

https://user-images.githubusercontent.com/42802102/148940990-ef9389c9-c2ab-4c25-b165-326e8f02d0dd.mp4

I think this is worth addressing, as anyone with a 4k+ monitor is going to see that screen size.

Reply:
Thanks. Noticed the first issue much earlier, completely forgot to fix that. 
The header card losing its fade is intentional ([context](https://hashintel.slack.com/archives/C02NW18QGUR/p1641822258008500)). Although the card still being cutoff at those screen sizes is a bug and I'll fix that.

Reply:
@teenoh It's fine for the card to lose it's fade, but I think the top-right and bottom-right corners should be rounded like their left-side counterparts when this is the case.

Reply:
Follow up [task](https://app.asana.com/0/1201519755227635/1201656851219256)